src/webroot/js/cake-core.js

```diff
@@ -0,0 +1,23 @@
+//$(document).ready(function () {
+var elements = document.getElementsByClassName('cake-core-postLink');
+
+var postLinkFunction = function (event) {
+    var formName = this.getAttribute("data-cake-core-form");
+    var confirmMessage = this.getAttribute('data-cake-core-confirm');
+    if (confirmMessage !== null && confirmMessage !== 'undefined') {
+        if (confirm(confirmMessage)) {
+            document[formName].submit();
+            return false;
+        }
+        return true;
+    }
+    document[formName].submit();
+    event.returnValue = false;
+    return false;
+};
+
+for (var i = 0; i < elements.length; i++) {
+    var element = elements[i];
+    element.onclick = postLinkFunction;
```

Comment:
element.addEventListener() ist well supported by Firefox and Chrome. By Opera since version 7.0 and by IE since 9.0. Is it ok for you to not support IE with version below 9.0?